packages/crawler/src/setup-crawler-container.ts

```diff
@@ -44,6 +46,23 @@ export function setupLocalCrawlerContainer(container: inversify.Container): inve
     return container;
 }
 
+export function setupCloudCrawlerContainer(container: inversify.Container): inversify.Container {
+    container.bind(iocTypes.CrawlerEngine).to(SimpleCrawlerEngine);
+    setupSingletonProvider(iocTypes.ApifyRequestQueueProvider, container, async (context: inversify.interfaces.Context) => {
```

Comment:
Do we not want the reporter factory for the cloud crawler?

Reply:
Not for the crawler itself, no. In the cloud crawl scenario, reports will be generated in the runner.